README.md

```diff
@@ -40,6 +40,7 @@ Consider the following example for a mariadb image:
 <service name="replace_using_package_version" mode="buildtime">
     <param name="file">kubic-mariadb-image.kiwi</param>
     <param name="regex">%%TAG%%</param>
```

Comment:
What is the TAG for? You are removing it from the image definitions below.

Reply:
Note this is an arbitrary marker, it could be anything. This is just an example. Anyway, I can use something closer to what is being included into the KIWI file.

Reply:
I think this is a bit confusing. Could you explain it a bit in the README?